app/controllers/admin/routes.py

```diff
@@ -150,11 +152,53 @@ def editEvent(eventId):
     return render_template("admin/createSingleEvent.html",
                             eventData = eventData,
                             allFacilitators = getAllFacilitators(),
-                            futureTerms = futureTerms,
+                            futureTerms=futureTerms,
                             isPastEvent = isPastEvent,
                             userHasRSVPed = userHasRSVPed,
                             isProgramManager = isProgramManager)
 
+@admin_bp.route('/eventsList/<eventId>/view', methods=['GET'])
```

Comment:
Let's try using the same controller method as the 'edit' page, since you are providing almost all of the same information.

Reply:
resolved